spring-boot-admin-server-ui/src/main/frontend/views/instances/env/refresh.vue

```diff
@@ -35,22 +51,62 @@ export default {
   computed: {
     instanceCount() {
       return this.application.instances.length;
+    },
+    refreshedPropertiesHtml() {
+      if (this.currentScope === 'instance') {
+        return '<ul class="properties-list">' + this.refreshedProperties[0].changedProperties.map(entry => ('<li>' + entry + '</li>')).join('') + '</ul>';
+      } else {
+        return '<ul class="properties-list">' + this.refreshedProperties.filter(property => property.changedProperties.length > 0).map(entry => ('<li>instanceId: ' + entry.instanceId + '<ul class="properties-list">' + entry.changedProperties.map(property => ('<li>' + property + '</li>')).join('') + '</ul>')).join('') + '</ul>';
+      }
+    }
+  },
+  data() {
+    return {
+      refreshedProperties: [],
+      isModalOpen: false,
+      currentScope: 'instance',
     }
   },
   methods: {
     refreshContext(scope) {
+      this.currentScope = scope;
       if (scope === 'instance') {
-        return this.instance.refreshContext();
+        return this.instance.refreshContext().then((response) => {
+          this.refreshedProperties = [{
+            instanceId: this.instance.id,
+            changedProperties: response.data,
+          }];
+          this.isModalOpen = response.data.length > 0;
+        });
       } else {
-        return this.application.refreshContext();
+        return this.application.refreshContext().then((response) => {
+          this.refreshedProperties = response.data.map(entry =>
+            (
+              {
+                instanceId: entry.instanceId,
+                changedProperties: JSON.parse(entry.body),
+              })
+          );
+          this.isModalOpen = this.refreshedProperties.some(props => { return props.changedProperties.length > 0; });
```

Comment:
Just skip return by removing braces. 

```suggestion
          this.isModalOpen = this.refreshedProperties.some(props =>  props.changedProperties.length > 0;);
```